src/frontend/AutoCorrectDialog.ui

```diff
@@ -1,228 +1,167 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
![autocorrect_mistakes](https://user-images.githubusercontent.com/9459891/43120741-2618d440-8f3d-11e8-95ab-66826eec201d.png)
That two text boxes are unequal in size. `Clear` button is also quite inconsistent.